code/modules/mob/living/simple_animal/abnormality/aleph/blue_star.dm

```diff
@@ -111,7 +111,9 @@
 		datum_reference.qliphoth_change(-1)
 	if(user.sanity_lost)
 		datum_reference.qliphoth_change(-1)
-	if(work_time > 40 SECONDS)
+
+	//Safety, just in case for some reason you can't hit the timer as it's inconsistent and hard to judge
+	if(work_time > 40 SECONDS && get_attribute_level(user, TEMPERANCE_ATTRIBUTE) > 100)
```

Comment:
Is this supposed to only go off while the agent took too long AND had a temperance ABOVE one hundred?

Reply:
This is a very weird part of code that makes no sense.

Reply:
Sounds like she meant to put an "OR"